src/Microsoft.Security.Utilities.Benchmarks/RE2RegexEngine.cs

```diff
@@ -18,13 +18,9 @@ public class RE2RegexEngine : IRegexEngine
 #endif
```

Comment:
I just noticed this is in benchmarks. Do tests use it?

Reply:
I see there are two copies. Can we use the same source in both benchmark and tests so that we are sure we're testing what we benchmark and vice versa.